src/frontend/.gitignore

```diff
@@ -2,3 +2,4 @@
 .env.staging
 node_modules/
 e2e/report
+*.png
```

Comment:
will this affect any other png files or just pngs in the top level directory?

Reply:
Removed. I was taking debugging and did not want these files ending up in repo